Libraries/acdriver/CMakeLists.txt

```diff
@@ -19,11 +19,11 @@ add_library(acdriver SHARED
 
 target_link_libraries(acdriver PUBLIC xcassets util plist ext)
 target_include_directories(acdriver PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/Headers")
-install(TARGETS acdriver DESTINATION usr/lib)
```

Comment:
Instead, push/set/pop `CMAKE_INSTALL_DEFAULT_COMPONENT_NAME` in the `Libraries/CMakeLists.txt` (or whatever calls `add_subdirectory`). This reduces the diff and is less nasty (though is action-at-a-distance).